Manual.md

```diff
@@ -0,0 +1,145 @@
+
+***
+
+## Introduction
+
+Each file under `packages/` is a recipe for `crew`, each recipe contains the instructions to compile from source and/or install as a binary package using `crew`. These package files are called `.rb`'s within this repository, but are otherwise know as ruby scripts.  
+
+The `.rbs`'s are ruby scripts called by `crew`<sup>(The Chromebrew package manager)</sup>. Each `.rb` must follow a explicit set of rules laid out by `crew`, these rules are provided within this Manual.
+
+***
+# Setup
+`crew` uses the following layout to install/build packages.
+
+1. `fetch` - During the fetch phase `crew` uses `curl --ssl`  to fetch the package
+2. `extract` - Once the the package has been fetched, `crew` extracts the package
+3. `self.prebuild` - If provided, only `sed` commands should be used during this stage
+	- During this stage `crew` `cd`'s into the extracted directory, and stays there until the `preinstall` stage 
```

Comment:
We currently do not have a `self.prebuild` section.  This needs to be added first.